### PR TITLE
Remove old image references.

### PR DESCRIPTION
**Pull request by felicianotech**

### CODE REVIEW

File: jekyll/_cci2/concepts.md [hunk 1]
```diff
@@ -181,7 +181,44 @@ jobs:
           # non-login shell with /bin/bash -eo pipefail option
           # by default.
 #...          
-```
+```          
+
+## Image
```
Comment (teesloane) [inline jekyll/_cci2/concepts.md:186]: I'm glad we're adding this to concepts.

File: jekyll/_cci2/executor-types.md [hunk 2]
```diff
@@ -203,42 +203,36 @@ Using the `machine` executor also means that you get full access to the Docker p
...
 
-**Note:** The default image for the machine executor is `circleci/classic:latest`.  If you don't specify an image, jobs will run on the default image - which is currently circleci/classic:201710-01 but may change in future.
-
-All images have common language tools preinstalled. Refer to the [specification script for the VM](https://raw.githubusercontent.com/circleci/image-builder/picard-vm-image/provision.sh) for more information.
-
-**Note:** The `image` key is not required on self-hosted installations of CircleCI Server (see example above) but if it is used, it should be set to: `image: default`.
-
-The following example uses the default machine image and enables [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching) (DLC) which is useful when you are building Docker images during your job or Workflow.
+The following example uses an image and enables [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching) (DLC) which is useful when you are building Docker images during your job or Workflow. **Note:** You must open a support ticket to have a CircleCI Sales representative contact you about enabling this feature on your account for an additional fee.
```
Comment (teesloane) [inline jekyll/_cci2/executor-types.md:220]:
```suggestion
The following example uses an image and enables [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching) (DLC) which is useful when you are building Docker images during your job or workflow. **Note:** You must open a support ticket to have a CircleCI Sales representative contact you about enabling this feature on your account for an additional fee.
```

**Review by teesloane — COMMENTED** [review]
Thanks for making this PR. Is it still in Draft?

**Comment by felicianotech** [comment]
It's ready now. I needed to add/update the Docker versions section as well.

File: jekyll/_cci2/building-docker-images.md [hunk 3]
```diff
@@ -80,6 +81,7 @@ jobs:
       # ... steps for building/testing app ...
 
       - setup_remote_docker:
+	      version: 19.03.12
```
Comment (teesloane) [inline jekyll/_cci2/building-docker-images.md:84]: I think indentation might be off here?

File: jekyll/_cci2/concepts.md [hunk 4]
```diff
@@ -181,7 +181,44 @@ jobs:
...
+
+An image is a packaged system that has the instructions for creating a running container.
```
Comment (teesloane) [inline jekyll/_cci2/concepts.md:188]:
```suggestion
An image is a packaged system that has instructions for creating a running container. 
```

File: jekyll/_cci2/concepts.md [hunk 5]
```diff
@@ -181,7 +181,44 @@ jobs:
           # non-login shell with /bin/bash -eo pipefail option
           # by default.
 #...          
-```
+```          
+
+## Image
+
+An image is a packaged system that has the instructions for creating a running container. 
+ The Primary Container is defined by the first image listed in [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file. This is where commands are executed for jobs using the Docker or machine executor. The Docker executor spins up a container with a Docker image. The machine executor spins up a complete Ubuntu virtual machine image. See [Choosing an Executor Type]({{ site.baseurl }}/2.0/executor-types/) document for a comparison table and considerations.
```
Comment (teesloane) [inline jekyll/_cci2/concepts.md:189]:
```suggestion
 The Primary Container is defined by the first image listed in a [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file. This is where commands are executed for jobs using the Docker or machine executor. The Docker executor spins up a container with a Docker image. The machine executor spins up a complete Ubuntu virtual machine image. See the [Choosing an Executor Type]({{ site.baseurl }}/2.0/executor-types/) document for a comparison table and considerations.
```

**Review by teesloane — CHANGES_REQUESTED** [review]

**Comment by teesloane** [comment]
> It's ready now. I needed to add/update the Docker versions section as well.

Awesome, thanks! Just put up a few suggestions and then should be good to merge.

File: jekyll/_cci2/building-docker-images.md (hunk 3 above)
Comment (felicianotech) [reply jekyll/_cci2/building-docker-images.md:84]: Checking

Comment (felicianotech) [reply jekyll/_cci2/building-docker-images.md:84]: Correct. There was a tab.

**Comment by felicianotech** [comment]
Updated with suggestions.

**Review by teesloane — APPROVED** [review]

**Comment by felicianotech** [comment]
Thanks for your help @teesloane.

**Pull request merged**